Gemfile

```diff
@@ -1,5 +1,9 @@
 source 'https://rubygems.org'
 
+# These were previously included in Ruby, but will be removed as of Ruby 3.4
+gem 'base64', '~> 0.2.0'
+gem 'bigdecimal', '~> 3.1.8'
+
```

Comment:
Technically, these shouldn't be here because they are dependencies required by our dependencies. There are issues about this raised in their repos though:

* safe_yaml: https://github.com/dtao/safe_yaml/issues/108
* liquid: https://github.com/Shopify/liquid/issues/1772